saleor/graphql/order/mutations/orders.py

```diff
@@ -514,3 +515,32 @@ class Meta:
         model = models.Order
         permissions = (OrderPermissions.MANAGE_ORDERS,)
         public = False
+
+
+class RequestInvoice(BaseMutation):
+    class Arguments:
+        order_id = graphene.Int(
+            required=True, description="ID of the order related to invoice."
```

Comment:
this is only a detail but required is True by default so no need to add it :)

Reply:
is it? if i remove it and re-generate the schema it's missing ! mark near this field

Reply:
@korycins Default is False ;)

Reply:
Whatever it is, I find it useful to have it explicitly defined.